### PR TITLE
Added aten::norm operations.

### PR DESCRIPTION
**Pull request by dlibenzi**

### CODE REVIEW

File: torch_xla/csrc/ops/ops.cpp [hunk 1]
```diff
@@ -415,6 +416,29 @@ NodePtr BroadcastTensors(tensorflow::gtl::ArraySlice<const Value> tensors) {
...
+  if (!p.has_value() || p->toDouble() == 2.0) {
+    NodePtr square = input * input;
+    NodePtr result = MakeNode<Sum>(square, dimensions, keepdim, dtype);
+    return Sqrt(result);
+  }
+  if (p->toDouble() == 1.0) {
+    // Nuclear Norm. Needs trace.
+  }
+  // Generic sum(x^p)^(1/p) norms.
+  XLA_ERROR() << "Norm kind not implemented: " << *p;
```
Comment (asuhan) [inline torch_xla/csrc/ops/ops.cpp]: Shouldn't we re-route to CPU in ATen instead of failing here?

Comment (dlibenzi) [reply torch_xla/csrc/ops/ops.cpp]: PR is incomplete. Just added the general form. Waiting for the nuclear one for the trace.

**Comment by dlibenzi** [comment]
Ready for review.
Spent quite some time chasing bad documentation.

File: torch_xla/csrc/ops/ops.cpp [hunk 2]
```diff
@@ -415,6 +416,42 @@ NodePtr BroadcastTensors(tensorflow::gtl::ArraySlice<const Value> tensors) {
...
+    dimensions = xla::util::Iota<xla::int64>(input.shape().rank());
+  }
+  if (!p.has_value() || p->toDouble() == 2.0) {
+    NodePtr square = input * input;
+    NodePtr result = MakeNode<Sum>(square, dimensions, keepdim, dtype);
+    return Sqrt(result);
+  }
+  double norm_value = p->toDouble();
+  if (norm_value == 1.0) {
+    // Contrary to documentation, norm(p=1) has nothing to do with traces and
```
Comment (asuhan) [inline torch_xla/csrc/ops/ops.cpp]: Isn't the nuclear norm the sum of singular values? I'm equally confused by the documentation, I guess we'll go with this.

Comment (dlibenzi) [reply torch_xla/csrc/ops/ops.cpp]: > Isn't the nuclear norm the sum of singular values? I'm equally confused by the documentation, I guess we'll go with this.

Oh, looking *at their code* it is clearer!
We need to trace(SVD(x)) it.
Doing it ...

**Pull request merged**